datacenterlight/templates/datacenterlight/landing_payment.html

```diff
@@ -163,12 +164,13 @@ <h5 class="membership-lead">Type: {{credit_card_data.cc_brand}}</h5>
                                             </p>
                                         {% endif %}
                                     {% endfor %}
-
+                                    {% comment %}
```

Comment:
Is this commented block intentional?

Reply:
Good catch. Well, `form` is no more used. So removing it.